src/Raven.Server/ServerWide/ServerStore.cs

```diff
@@ -422,10 +422,10 @@ private void GenerateAuthenticationKeyPairs(TransactionOperationContext ctx)
 
             using (var tx = ctx.OpenWriteTransaction())
             {
-                PutSecretKey(ctx, "Box PK", BoxPublicKey, overwrite:true, cloneKey:true);
-                PutSecretKey(ctx, "Box SK", BoxSecretKey, overwrite: true, cloneKey: true);
-                PutSecretKey(ctx, "Sign PK", AuthPublicKey, overwrite: true, cloneKey: true);
-                PutSecretKey(ctx, "Sign SK", SignSecretKey, overwrite: true, cloneKey: true);
+                PutSecretKey(ctx, "Raven/Box/Public", BoxPublicKey, overwrite:true, cloneKey:true);
```

Comment:
Note that @talweiss1982 issue here is important, we can't really use `PutSecretKey` here, since this will cause us to overwrite the value when we are installing the snapshot.

First, the Box keys don't need to be persistent, only the sign keys.
Second, we need to store them _outside_ of the state machine, since they are purely local affairs.